lib/components/collapse.vue

```diff
@@ -24,19 +24,32 @@
 
         data() {
             return {
-                show: this.showOnInit
+                show: this.visible
             };
         },
 
         computed: {
             classObject() {
+                // console.log('class object evaluated')
                 return {
                     'navbar-collapse': this.isNav,
                     show: this.show
                 };
             }
         },
 
+        model: {
+            prop: 'visible',
+        },
+
+        watch: {
+            visible(newVal, oldVal){
+                if (newVal !== this.show) {
+                    this.toggle()
```

Comment:
I thought about that, but then I would have to re-implement the emitState logic. Thoughts?